src/box/xrow.c

```diff
@@ -2063,6 +2072,15 @@ xrow_decode_replication_request(const struct xrow_header *row,
 				return -1;
 			}
 			break;
+		case IPROTO_INSTANCE_NAME:
+			if (req->instance_name == NULL)
+				goto skip;
+			if (xrow_decode_hostname(&d, req->instance_name) != 0) {
+				xrow_on_decode_err(row, ER_INVALID_MSGPACK,
+						   "instance NAME");
```

Comment:
Because all the other names in this function ARE ALSO CAPS: `"invalid VCLOCK"`, `"invalid VERSION"`, `"invalid REPLICA_ANON flag"`, `"invalid ID_FILTER"`. I didn't want to redo them all.

Why they are done like this is understandable. I think the authors wanted to emphasise that these are invalid fields of a message. Not invalid attributes of the whole message itself or even of this instance. Although I do think the messages should just use format like `"invalid field \"version\""`, `"invalid field \"id_filter\""`, etc.

I also see now that my messages are inconsistent. I used `"replicaset name"` without caps. By this logic I should make them whole caps, like `"invalid INSTANCE_NAME"`, `"invalid REPLICASET_NAME"`. Will do that.

Unless you want me to fix all those places to something like `"invalid field \"<name>\""` as a preparatory commit.

Reply:
> Because all the other names in this function ARE ALSO CAPS

Missed that, sorry. Then it's fine. Let's leave it as is.